app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ApplicationRecord
+  belongs_to :users
+  has_many :likes, :comments
+
+  def update_user_post_counter
+    users.posts_counter += 1
+  end
+
+  def return_most_recent_comments
+    comments.last(5)
+  end
+end
```

Comment:
- [ ] Posts have many comments, please implement them.
- [ ] here you can use class_name: 'User' instead of :users